docs/getting_started.rst

```diff
@@ -147,11 +146,30 @@ Automatic Installation on Windows
 
 // TODO(hoatle): add this
 
-.. _manual-installation-on-windows:
 
 Manual Installation on Windows
 ------------------------------
```

Comment:
Please use `chocolatey` so that users can copy + paste commands to install required packages.

Reply:
applied, pls check again